rpa-manager/modules/ROOT/pages/processautomation-manage.adoc

```diff
@@ -1,91 +1,94 @@
+* To transition a project to another phase, you must have the needed permissions as a member of the <<projectdata-processteam, process team>>.
 
 == Create an Automation project
 
-When a project manager clicks the icon *Start automation* in the *My Backlog* view of the *My RPA* module, he creates a process automation project assigned to a category in the *Process Management* view of the *Process Automation* module.
+When a project manager clicks the icon *Start automation* for a process in the *My Backlog* view of the *My RPA* module, RPA Manager creates a project for the automation of the process.
 
-If you already have evaluated a process with a third-party tool, you can create an automation project directly inside a category:
+To automate a process already evaluated with a third-party tool, you can create an automation project inside a category directly in the *Process Automation* module:
 
-. In a category in the *Process Management* view of the *Process Automation* module, click *Create*.
-. Complete the form *Create process*:
-// team permissions included in form
-[[process-team-permissions]]
+. Open the *Project Management* view of the *Process Automation* module.
+. Click on the name of the category in which you want to create the new project.
+. In the *<category name>* view, click *Create*.
+. [[form-createprocess]] Complete the form *Create process*:
 +
 include::rpa-manager::partial$form-automationproject.adoc[]
 . Click *Save*.
 
-== Copy an Automation project
-
-When a project manager clicks the icon *Start automation* in the *My Backlog* view of the *My RPA* module, he creates a process automation project assigned to a category in the *Process Management* view of the *Process Automation* module.
+RPA Manager returns to the *<category name>* view. Your new project is sorted into the view according to your workspace settings.
 
-If you already have evaluated a process with a third-party tool, you can copy and edit a similar automation project directly inside a category.
-
-The copy contains all information from the original project including the most recent revision of all workflows with the exception of the bot assignments to test and productive configurations, and the history of the workflows.
+== Copy an Automation project
 
-To complete copying of the project, first transfer the project to the *Build* phase. Then open it in the MuleSoft RPA Builder. The project then receives its own identification values. The copied project can only transition to the *Test* phase after it has been uploaded from the RPA Builder to the RPA Manager.
+A copy of an automation project contains all information from the original project including the most recent revision of all workflows with the exception of the bot assignments to test and productive configurations.
 
-Copy an existing similar project to save time:
+Copy a project to automate a similar process faster:
```

Comment:
```suggestion
Copy a project to automate a similar more quickly:
```

Reply:
@dhirsch1 there is a difference between process and project ( see above). Therefor I included the word 'process': '
Copy a project to automate a similar process more quickly:'